sdk/postgresql/Azure.ResourceManager.PostgreSql/api/Azure.ResourceManager.PostgreSql.netstandard2.0.cs

```diff
@@ -449,717 +487,708 @@ public partial class ConfigurationData : Azure.ResourceManager.Models.ResourceDa
+        public bool Equals(Azure.ResourceManager.PostgreSql.FlexibleServers.Models.PostgreSqlFlexibleServerVersion other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.PostgreSql.FlexibleServers.Models.PostgreSqlFlexibleServerVersion left, Azure.ResourceManager.PostgreSql.FlexibleServers.Models.PostgreSqlFlexibleServerVersion right) { throw null; }
+        public static implicit operator Azure.ResourceManager.PostgreSql.FlexibleServers.Models.PostgreSqlFlexibleServerVersion (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.PostgreSql.FlexibleServers.Models.PostgreSqlFlexibleServerVersion left, Azure.ResourceManager.PostgreSql.FlexibleServers.Models.PostgreSqlFlexibleServerVersion right) { throw null; }
+        public override string ToString() { throw null; }
     }
-    public partial class VirtualNetworkSubnetUsageParameter
+    public partial class PostgreSqlFlexibleServerVirtualNetworkSubnetUsageParameter
     {
-        public VirtualNetworkSubnetUsageParameter() { }
-        public string VirtualNetworkArmResourceId { get { throw null; } set { } }
+        public PostgreSqlFlexibleServerVirtualNetworkSubnetUsageParameter() { }
+        public Azure.Core.ResourceIdentifier VirtualNetworkArmResourceId { get { throw null; } set { } }
     }
-    public partial class VirtualNetworkSubnetUsageResult
+    public partial class PostgreSqlFlexibleServerVirtualNetworkSubnetUsageResult
     {
-        internal VirtualNetworkSubnetUsageResult() { }
-        public System.Collections.Generic.IReadOnlyList<Azure.ResourceManager.PostgreSql.FlexibleServers.Models.DelegatedSubnetUsage> DelegatedSubnetsUsage { get { throw null; } }
+        internal PostgreSqlFlexibleServerVirtualNetworkSubnetUsageResult() { }
+        public System.Collections.Generic.IReadOnlyList<Azure.ResourceManager.PostgreSql.FlexibleServers.Models.PostgreSqlFlexibleServerDelegatedSubnetUsage> DelegatedSubnetsUsage { get { throw null; } }
         public Azure.Core.AzureLocation? Location { get { throw null; } }
         public string SubscriptionId { get { throw null; } }
     }
 }
 namespace Azure.ResourceManager.PostgreSql.Models
 {
     [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
-    public readonly partial struct AdministratorType : System.IEquatable<Azure.ResourceManager.PostgreSql.Models.AdministratorType>
+    public readonly partial struct IdentityType : System.IEquatable<Azure.ResourceManager.PostgreSql.Models.IdentityType>
```

Comment:
prepend `PostgreSql` ?

Reply:
This will be replaced by common type after https://github.com/Azure/autorest.csharp/pull/2429 is in.